frontend/src/components/commons/FormInputs/Type.js

```diff
@@ -0,0 +1,55 @@
+import Grid from '@material-ui/core/Grid';
+import Tooltip from '@material-ui/core/Tooltip';
+import TextField from '@material-ui/core/TextField';
+import MenuItem from '@material-ui/core/MenuItem';
+import React, { useState } from 'react';
+
+const types = [
+  {
+    value: 'cityscape',
+    label: 'Stadt',
+  },
+  {
+    value: 'harbour',
+    label: 'Hafen',
+  },
+  {
+    value: 'landscape',
+    label: 'Landschaft',
+  },
+  {
+    value: 'night',
+    label: 'Nachtaufnahme',
+  },
+];
+
+export default function Type({ placeData }) {
+  const [type, setType] = useState('cityscape');
+
+  return (
+    <Grid item xs={12} sm={6}>
+      <Tooltip title={'Bitte auswählen'} arrow>
+        <TextField
+          select
```

Comment:
this `select` is what makes it a drowdown selector, and lines 42-45 are used for displaying the options that are given in line 7 ff.